website/docs/components/toast/partials/guidelines/overview.md

```diff
@@ -1 +1 @@
-A Toast is an element intended for messages resulting from a user’s actions. It typically disappears after a set interval or due to a user dismissing it. For system-generated messages, see the [Alert](/components/alert) component.
\ No newline at end of file
+Toasts deliver messages resulting from a user's actions. They typically disappear after a set interval or upon user dismissal, offering a non-intrusive way to provide feedback. While [Alerts](/components/alert) handle system-generated messages or critical information, Toasts suit less urgent feedback, such as indicating a user-initiated process's success or ongoing status.
```

Comment:
Providing feedback based on our earlier conversation. I removed the second half of this section to:

1. reduce confusion around terms such as "system-generated"
2. remove duplicative information (info is also presented in the when to use/when not to use section)

```suggestion
Toasts deliver messages to communicate an ongoing process or as a result of a user's actions. They typically disappear after a set interval or upon user dismissal, offering a non-intrusive way to provide feedback. 
```